recipes/darr/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "darr" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 183886142de227c8179409b8c69a5dfec0c74d467f9aa78be8224070961cad1b
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy >=1.12
+
+test:
+  imports:
+    - darr
+
+about:
+  home: https://github.com/gbeckers/darr
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A Python science library for memory-mapped numeric arrays, based on a format that is self-explanatory and tool-independent.'
```

Would wrap this as well.

```suggestion
  summary: >-
    A Python science library for memory-mapped numeric arrays, based on
    a format that is self-explanatory and tool-independent.
```